app/views/submit_form.scala.html

```diff
@@ -21,21 +21,37 @@
 @import templates.formatters.dates.displayDate
 @import templates.formatters.navigationBreadcrumb
 @import templates.formatters.dates.displayDateRange
-@import play.api.libs.json.Json
 
 @(periodKey: String,
-  clientName: Option[String],
-  flatRateScheme: Boolean,
-  obligations: VatObligations,
-  form: Form[NineBoxModel],
-  isAgent: Boolean)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+        clientName: Option[String],
+        flatRateScheme: Boolean,
+        obligation: VatObligation,
+        form: Form[SubmitVatReturnModel],
```

Comment:
The FRS and obligation data is now passed in twice to the view (as separate parameters and within the form model). Can they not be removed from the form entirely?